### DISCUSSION TITLE
How can I get the license information box for WTFPL

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45452] [Repositories] [answered]
### Select Topic Area

Question

### Body

I'm kinda new with GitHub repositories and I'm starting to add licenses to my small projects and learning how to use them. There was no license template for WTFPL so I did what the webpage for the license said and copy pasted the texts there into my `LICENSE` file and put in my name and e-mail address. But there is no license information box like when I used templates.
Also when I say license information boxes I mean these:

```
<repository> is licensed under the
**MIT License**
A short and simple permissive license with conditions only requiring preservation of copyright and license notices. Licensed works, modifications, and larger works may be distributed under different terms and without source code.
```

I'm a beginner so I'm sorry if this is a stupid question. Thanks!

**Comment by airtower-luna** [comment]
GitHub uses [Licensee](https://github.com/licensee/licensee) to determine the project license. The documentation over there describes [what it looks at](https://github.com/licensee/licensee/blob/main/docs/what-we-look-at.md). Either the license you're using isn't known to Licensee at all (WTFPL is not listed on https://choosealicense.com/licenses/), or if it is the `LICENSE` file you added is too different from the template.

**Reply by ghost** [reply]
Repositories like https://github.com/rpherrera/WTFPL/blob/master/LICENSE have the license information box so I think it is known to Licensee. My `LICENSE` file could also be too different from the template, not really sure. Should I change the license? Thanks for your response.

**Comment by spenserblack** [comment, marked as answer]
[This license](https://github.com/pizzavitdit/setupwin/blob/a7f018b321514f95ceb320d7f897ca6d278a0cc6/LICENSE)?

the version in your repo is pretty different than [the vendored license](https://github.com/licensee/licensee/blob/bd644a99e1892e88eca08c7dfeaa7fa834fc584e/vendor/choosealicense.com/_licenses/wtfpl.txt), since it has all that text after line 13 (starting at the *second* copyright header).
Without looking at the source code for licensee, I'm going to hazard a guess that there's a minimum similarity threshold that the license must meet, and all your extra text puts your license under that threshold. (Thanks @airtower-luna for pointing out licensee!)

**Reply by ghost** [reply]
Thanks!
